CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Braintree Android Drop-In Release Notes
 
+## unreleased
+* Bump card-form version to 5.0.0
```

Comment:
we should add a breaking changes section and include that support for Card.io was dropped

Reply:
👍